CMakeLists.txt

```diff
@@ -406,6 +430,13 @@ if (IS_64BIT_BUILD EQUAL 1)
   endif (CLR_CMAKE_PLATFORM_UNIX_TARGET_AMD64)
   add_definitions(-D_TARGET_AMD64_=1)
   add_definitions(-DDBG_TARGET_AMD64)
+elseif (CLR_CMAKE_PLATFORM_ARCH_ARM)
+  if (CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
+    add_definitions(-DDBG_TARGET_ARM_UNIX)
+  endif (CLR_CMAKE_PLATFORM_UNIX_TARGET_ARM)
+  add_definitions(-D_TARGET_ARM_=1)
+  add_definitions(-DDBG_TARGET_ARM)
+  add_definitions(-DSTUBLINKER_GENERATES_UNWIND_INFO)
```

Comment:
It may be better to go with STUBLINKER_GENERATES_UNWIND_INFO undefined - same as what we do on Linux AMD64. You can make it work either way, but I expect that going with STUBLINKER_GENERATES_UNWIND_INFO is going to be more work.

Getting this right will be important for managed exception handling. You can ignore it for "hello world".

STUBLINKER_GENERATES_UNWIND_INFO is defined in vm\stublink.h for all other cases. If it is really needed, it may be better to define it for this case too.